types/three/tsconfig.json

```diff
@@ -71,6 +71,8 @@
         "test/utils/utils-webgl-portal.ts",
         "test/utils/utils-webgl-marchingcubes.ts",
         "test/webxr/webxr-ar-lighting.ts",
-        "test/webxr/webxr-vr-handinput-cubes.ts"
+        "test/webxr/webxr-vr-handinput-cubes.ts",
+        "test/nodes/shadernode-code-samples.ts",
```

Comment:
These should be in alphabetical order please!

Reply:
Rearranged.